website/docs/docs/building-a-dbt-project/metrics.md

```diff
@@ -65,39 +73,37 @@ metrics:
       - field: is_paying
         value: true
 
-    meta: {}
+    meta: {team: Finance}
 ```
 
 </File>
 
-### Available properties
-
-| Field       | Description                                                 | Example                         | Required? |
-|-------------|-------------------------------------------------------------|---------------------------------|-----------|
-| name        | A unique identifier for the metric                          | new_customers                   | yes       |
-| model       | The dbt model that powers this metric                       | dim_customers                   | yes       |
-| label       | A short for name / label for the metric                     | New Customers                   | no        |
-| description | Long form, human-readable description for the metric        | The number of customers who.... | no        |
-| type        | The type of calculation to perform when evaluating a metric | count_distinct                  | yes       |
-| sql         | The expression to aggregate/calculate over                  | user_id                         | yes       |
-| timestamp   | The time-based component of the metric                      | signup_date                     | yes       |
-| time_grains | One or more "grains" at which the metric can be evaluated   | [day, week, month]              | yes       |
-| dimensions  | A list of dimensions to group or filter the metric by       | [plan, country]                 | no        |
-| filters     | A list of filters to apply before calculating the metric    | See below                       | no        |
-| meta        | Arbitrary key/value store                                   | {team: Finance}                 | no        |
-
-### Why define metrics?
-
-**Use metric specifications in downstream tools.** Metrics are available to dbt's compilation context via the [`graph.metrics` variable](graph). They are included in [the manifest artifact](manifest-json) for downstream metadata consumption.
+### Metrics properties
 
-**See and select dependencies.** As with exposures, it's possible to see everything that rolls up into a metric (`dbt ls -s +metric:*`), and visualize them in [dbt documentation](documentation).
+You can see how these properties are used in the [example YAML file](#declaring-a-metric).
 
-<Lightbox src="/img/docs/building-a-dbt-project/dag-metrics.png" title="Metrics appear as pink nodes in the DAG (for now)"/>
+| Field       | Description                                               | Example                           | Required? |
```

Comment:
I made some subtle changes only:
* Capitalized Yes/No
* Paralleled content from the YAML example.
* Shortened the descriptions